ush/setup.py

```diff
@@ -842,43 +820,23 @@ def get_location(xcs, fmt, expt_cfg):
     # -----------------------------------------------------------------------
     #
     grid_gen_method = workflow_config["GRID_GEN_METHOD"]
+    grid_config = expt_config["task_make_grid"]
+
+    grid_settings = grid_config["grid_settings"]
     if grid_gen_method == "GFDLgrid":
-        # where does user-defined grid get defined? config is empty
```

Comment:
This section is where we gain the benefits of isolating "grid_settings" as the input params to these functions.